bin/move-mysql-vip-to-mysql-master.sh

```diff
@@ -78,11 +78,11 @@ if [[ "$VIP_USED" ]] ; then
     # Unmount the VIP on the current VIP holder 
     if [[ "$SLAVE_DB" == "$(hostname)" ]] ; then
         [[ "$DEBUG" ]] && echo "Local ifdown"
-        ifdown "${MYSQL_VIP_IFNAME}:1"
+        sudo nmcli con mod "${MYSQL_VIP_IFNAME}" -ipv4.addresses "${MYSQL_VIP_IPADDR}/${MYSQL_VIP_CIDR_NETMASK}" && sudo nmcli con up "${MYSQL_VIP_IFNAME}"
     else
         # If not, we run it via SSH
         [[ "$DEBUG" ]] && echo "Remote ifdown"
-        sudo -u mysql ssh "$SLAVE_DB" -- sudo ifdown "${MYSQL_VIP_IFNAME}:1"
+        sudo -u mysql ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${SLAVE_DB} -- "sudo nmcli con mod ${MYSQL_VIP_IFNAME} -ipv4.addresses ${MYSQL_VIP_IPADDR}/${MYSQL_VIP_CIDR_NETMASK} && sudo nmcli con up ${MYSQL_VIP_IFNAME}"
```

Comment:
Is it what you wanted to replace "ifdown" with "con up" ?

Reply:
idem as previous